first.c

```diff
@@ -0,0 +1,112 @@
+
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+
+void removeWhitespaces(char *exp)
+{
+    char *p = exp;
+    char *q = exp;
+    while (*q != '\0')
+    {
+        if (!isspace(*q))
+        {
+            *p++ = *q;
+        }
+        q++;
+    }
+    *p = '\0';
+};
+
+bool checkValidity(char *exp)
+{
+    char oprs[4] = {'+', '-', '*', '/'};
+    for (int i = 0; exp[i] != '\0'; i++)
+    {
+        if (!(exp[i] >= '0' && exp[i] <= '9') && !(exp[i] == '+' || exp[i] == '*' || exp[i] == '-' || exp[i] == '/'))
+        {
+            return false;

```

Comment:
multiple return statements? Method should have a single return statement

Reply:
I will change the method to have only one return statement.